ArduCopter/crash_check.cpp

```diff
@@ -178,6 +178,12 @@ void Copter::parachute_check()
         return;
     }
 
+    // pass is_flying to parachute library
+    parachute.set_is_flying(!ap.land_complete);
+
+    // pass sink rate to parachute library
+    parachute.set_sink_rate(-inertial_nav.get_velocity_z() * 0.01);
```

Comment:
0.01f ?

Reply:
The "f" isn't actually required so Tridge often doesn't include it.  I tend to include it but we shouldn't insist that it is present.  In any case, it's Copter so I've added the "f" but again, it's not required.